### ISSUE TITLE
Pinfinder keeps telling me none.

### ISSUE THREAD

**Issue by EmLemer** [gwatts/pinfinder#125]
I think I did everything right. I have a IOS 11 and backed my phone to iTunes

**Comment by gwatts** [comment]
@EmLemer did you turn on encryption for the backup?

**Comment by EmLemer** [comment]
Yes I am pretty sure I did. There isn't really an option to change it and has it automatically turned on to encrypted. Do you have any way I can make sure it is though?

**Comment by gwatts** [comment]
you should see if on the backup screen:
![image](https://github.com/gwatts/pinfinder/assets/334535/e517dcc3-98ba-425f-afb6-43b7abc3c136)
